client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -84,7 +85,7 @@ class TaxonomyManagerListItem extends Component {
 					<Gridicon
```

Comment:
Why don't we use [`<EllipsisMenu />`](https://github.com/Automattic/wp-calypso/tree/master/client/components/ellipsis-menu) instead of recreating it manually?